config/ax_code_coverage.m4

```diff
@@ -222,7 +222,7 @@ CODE_COVERAGE_GENHTML_OPTIONS ?= $(CODE_COVERAGE_GENHTML_OPTIONS_DEFAULT)
 
 # Add any folders you want to ignore here
 # Ignore tmp and tests themselves
-CODE_COVERAGE_IGNORE_PATTERN ?= "/tmp/*" "*/tests/*"  
+CODE_COVERAGE_IGNORE_PATTERN ?= "/tmp/*" "*/tests/*" "*/contrib/zstd/*"
```

Comment:
@c0d3z3r0 
Please change this to the new single lib file(s), this adds the codecov ignore.

Reply:
good catch, thx